developerguide/secure-tunneling.md

```diff
@@ -1,5 +1,5 @@
 # AWS IoT secure tunneling<a name="secure-tunneling"></a>
 
-When devices are deployed behind restricted firewalls at remote sites, you need a way to gain access to those devices for troubleshooting, configuration updates, and other operational tasks\. Use secure tunneling to establish bidirectional communication to remote devices over a secure connection that is managed by AWS IoT\. secure tunneling does not require updates to your existing inbound firewall rule, so you can keep the same security level provided by firewall rules at a remote site\.
+When devices are deployed behind restricted firewalls at remote sites, you need a way to gain access to those devices for troubleshooting, configuration updates, and other operational tasks\. Use secure tunneling to establish bidirectional communication to remote devices over a secure connection that is managed by AWS IoT\. Secure tunneling does not require updates to your existing inbound firewall rules, so you can keep the same security level provided by firewall rules at a remote site\.
 
-For example, a sensor device located at a factory that is a couple hundred miles away is having trouble measuring the factory temperature\. You can use secure tunneling to open and quickly start a session to that sensor device\. After you have identified the problem \(for example, a bad configuration file\), you can reset the file and restart the sensor device through the same session\. Compared to a more traditional troubleshooting \(for example, sending a technician to the factory to investigate the sensor device\), secure tunneling decreases incident response and recovery time and operational costs\.
\ No newline at end of file
+For example, a sensor device located at a factory that is a couple hundred miles away is having trouble measuring the factory temperature\. You can use secure tunneling to open and quickly start a session to that sensor device\. After you have identified the problem \(for example, a bad configuration file\), you can reset the file and restart the sensor device through the same session\. Compared to a more traditional troubleshooting \(for example, sending a technician to the factory to investigate the sensor device\), secure tunneling decreases incident response and recovery time and operational costs\.
```

Comment:
No changes in this paragraph, not dure why it shows a diff

Reply:
Thanks! I'll put this in the next build.